src/ADC_SPI.cpp

```diff
@@ -12,7 +12,7 @@
  */
 ADC_SPI::ADC_SPI()
 {
-	init(DEFAULT_SPI_CS, DEFAULT_SPI_SPEED);
+	// init(DEFAULT_SPI_CS, DEFAULT_SPI_SPEED);
 }
```

Comment:
running "init" in the constructor would crash teensy b/c SPI.begin() gets called in the pedalHandler constructor and not in a function. this works